jpa/eclipselink.jpa.test.maven/pom.xml

```diff
@@ -0,0 +1,2066 @@
+        </testResources>
+
+        <plugins>
+            <!--Required for EL Weaving (-javaagent:${org.eclipse.persistence:org.eclipse.persistence.jpa:jar})-->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <!--Resolve dependencies into Maven properties like ${org.eclipse.persistence:org.eclipse.persistence.jpa:jar} for JPA module-->
+                    <execution>
+                        <id>get-test-classpath-to-properties</id>
+                        <goals>
+                            <goal>properties</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <!--Required due filtering applied to org.eclipse.persistence.testing.framework.server.TestRunnerBean java class.-->
+            <plugin>
+                <groupId>com.google.code.maven-replacer-plugin</groupId>
+                <artifactId>replacer</artifactId>
+                <executions>
+                    <execution>
+                        <id>filter-test-src-TestRunnerBean</id>
+                        <phase>generate-test-sources</phase>
+                        <goals>
+                            <goal>replace</goal>
+                        </goals>
+                        <configuration>
+                            <file>${project.basedir}/${integration.test.directory}/java-templates/org/eclipse/persistence/testing/framework/server/TestRunnerBean.java</file>
+                            <outputFile>${project.build.directory}/generated-test-sources/org/eclipse/persistence/testing/framework/server/TestRunnerBean.java</outputFile>
+                            <replacements>
+                                <replacement>
+                                    <token>@puName@</token>
+                                    <value>${puName}</value>
+                                </replacement>
+                            </replacements>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <!--Generate some classes required for test by EclipseLink Canonical Model Processor.
+                Adds to the compiler (test scope) sources root path: .../target/generated-test-sources -->
+            <plugin>
+                <groupId>org.bsc.maven</groupId>
+                <artifactId>maven-processor-plugin</artifactId>
+                <configuration>
+                    <processors>
+                        <processor>org.eclipse.persistence.internal.jpa.modelgen.CanonicalModelProcessor</processor>
+                    </processors>
+                    <sourceDirectory>${project.basedir}/${integration.test.directory}/java/</sourceDirectory>
+                    <outputDirectory>${project.build.directory}/generated-test-sources</outputDirectory>
```

Comment:
this should go to sth like '${project.build.directory}/generated-test-sources/modelgen' instead